fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -147,7 +149,10 @@ void writeResponse(final FcrepoResponse response, final File file)
 
         final List<URI> describedby = response.getLinkHeaders("describedby");
         for (final Iterator<URI> it = describedby.iterator(); describedby != null && it.hasNext(); ) {
-            exportDescription(it.next());
+            exportDescription(response.getUrl(), it.next());
+            if (it.hasNext()) {
+                throw new RuntimeException("fcrepo does not produce multiple describedby headers.");
```

Comment:
I had something very similar to this when I was first handling binary descriptions, but @awoods pointed out that Fedora (or other LDP impls) might have multiple binary descriptions.  So that's why I used the description URI instead of the binary URI to build the filename.  My thinking was that there could be multiple descriptions for a binary, but each description would need to have its own URI, so therefore could be used to build a unique filename.